src/main/java/com/revature/controllers/MetricsController.java

```diff
@@ -41,64 +40,62 @@
 	SearchService searchService;
 	@Autowired
 	TimegraphService timegraphService;
-	
+
 	@PostMapping("/{timeFrame}")
-	public MetricsData getMetrics(@PathVariable("timeFrame") long timeRange, 
-								  @RequestBody Map<String, Object> filter) {		
+	public MetricsData getMetrics(@PathVariable("timeFrame") long timeRange, @RequestBody Map<String, Object> filter) {
```

Comment:
This method needs refactoring.
- why is it using a post request
- why are the filter parameters being passed into requestbody rather than as request parameters? map directly into an array or list instead of using StringTokenizer
- logic should be refactored (e.g. lines 62-69 have contradictory conditions that result in the same thing) and moved to a service method

Reply:
This doesn't need to be fixed in this pull request, as it's not really in scope. I'm not sure why there need be changes to this file for this pull request though. Perhaps handle this elsewhere.

Reply:
Hi @carolynrehm , most of the changes here are from IDE formatting. I changed three things. First, the StringTokenizer initialization because it was missing a null check which leads to an unreachable branch for testing at line 62. Second I removed the unchecked suppression since it was not necessary. Third, I removed the null check for idsIn since it will always get initialized at line 48. What I failed to realize was that the branches in line 62 were exactly the same. So I will remove the unnecessary if/else.

Reply:
Gotcha. Definitely some fixes that we can make but can we keep it out of this pull request just because it's not related to testing?

Reply:
Good to know, I will try not to modify too many files in the future.